sklearn/utils/__init__.py

```diff
@@ -506,3 +506,41 @@ def indices_to_mask(indices, mask_length):
     mask[indices] = True
 
     return mask
+
+
+def get_chunk_n_rows(row_bytes, max_n_rows=None,
+                     working_memory=None):
+    """Calculates the number of rows that fit in working_memory
+
+    Parameters
+    ----------
+    row_bytes : int
+        The number of bytes consumed by each row of expected output
+        from some function being applied to each chunk.
```

Comment:
Just reading this docsrting it's unclear what function it could be. Maybe just refer to the array chunk size?

Reply:
I've tried to make this more precise.